crates/ra_mbe/src/syntax_bridge.rs

```diff
@@ -118,6 +118,69 @@ impl TokenMap {
     }
 }
 
+/// Returns the textual content of a doc comment block as a quoted string
+/// That is, strips leading `///` (or `/**`, etc)
+/// and strips the ending `*/`
+/// And then quote the string, which is needed to convert to `tt::Literal`
+fn doc_comment_text(comment: &ast::Comment) -> SmolStr {
+    use ast::AstToken;
+
+    let prefix_len = comment.prefix().len();
+    let mut text = &comment.text()[prefix_len..];
+
+    // Remove ending "*/"
+    if comment.kind().shape == ast::CommentShape::Block {
+        text = &text[0..text.len() - 2];
+    }
+
+    // Quote the string
+    // Note that `tt::Literal` expect an escaped string
+    let text = format!("\"{}\"", text.escape_default());
```

Comment:
I actually think that `format!("{:?}", text.escape_default())` is what we need here

Reply:
Fixed.